platform.mk

```diff
@@ -385,11 +385,11 @@ PRODUCT_PACKAGES += \
     android.hardware.bluetooth@1.0-service-qti
 
 PRODUCT_PROPERTY_OVERRIDES += \
-    persist.vendor.qcom.bluetooth.soc=hastings
+    persist.vendor.qcom.bluetooth.soc=hamilton
 
 # Legacy BT property (will be removed in S)
 PRODUCT_PROPERTY_OVERRIDES += \
-    vendor.qcom.bluetooth.soc=hastings
+    vendor.qcom.bluetooth.soc=hamilton
```

Comment:
Did this ever actually get removed? 😂

Reply:
it completely depends on BSP that's why I kept this property. its fate completely depends on Alin:)
```
pdx234:/ # grep -rnw /odm -e "persist.vendor.qcom.bluetooth.soc"                                                                                                                                                                           
Binary file /odm/lib64/hw/android.hardware.bluetooth@1.0-impl-qti.so matches
Binary file /odm/lib/hw/android.hardware.bluetooth@1.0-impl-qti.so matches
pdx234:/ # grep -rnw /odm -e "vendor.qcom.bluetooth.soc"                                                                                                                                                                                   
Binary file /odm/lib64/hw/android.hardware.bluetooth@1.0-impl-qti.so matches
Binary file /odm/lib64/libbthost_if.so matches
Binary file /odm/lib64/libbthost_if.so matches
Binary file /odm/lib/hw/android.hardware.bluetooth@1.0-impl-qti.so matches
Binary file /odm/lib/libbthost_if.so matches
Binary file /odm/lib/libbthost_if.so matches
```

Reply:
that's the BSP config

Reply:
Maybe we should adjust the comment then :slightly_smiling_face: